src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -686,11 +680,32 @@ function MoneyRequestConfirmationList({
                 {button}
             </>
         );
-    }, [isReadOnly, iouType, selectedParticipants.length, confirm, bankAccountRoute, iouCurrencyCode, policyID, splitOrRequestOptions, formError, styles.ph1, styles.mb2]);
+    }, [isReadOnly, iouType, selectedParticipants.length, confirm, bankAccountRoute, iouCurrencyCode, policyID, splitOrRequestOptions, formError, styles.ph1, styles.mb2, isTypeSplit]);
 
     // An intermediate structure that helps us classify the fields as "primary" and "supplementary".
     // The primary fields are always shown to the user, while an extra action is needed to reveal the supplementary ones.
     const classifiedFields = [
+        {
+            item: (
+                <MenuItem
+                    key={translate('moneyRequestConfirmationList.paidBy')}
+                    label={translate('moneyRequestConfirmationList.paidBy')}
+                    interactive={!isPolicyExpenseChat && !isReadOnly}
+                    description={payeePersonalDetails.login ?? ReportUtils.getDisplayNameForParticipant(payeePersonalDetails.accountID)}
+                    title={payeePersonalDetails.displayName ?? ReportUtils.getDisplayNameForParticipant(payeePersonalDetails.accountID)}
+                    icon={payeeIcons}
+                    onPress={() => {
+                        Navigation.navigate(
+                            ROUTES.MONEY_REQUEST_STEP_SPLIT_PAYER.getRoute(action, iouType, transaction?.transactionID ?? '', reportID, Navigation.getActiveRouteWithoutParams()),
+                        );
+                    }}
+                    shouldShowRightIcon={!isPolicyExpenseChat && !isReadOnly}
+                    titleWithTooltips={payeePersonalDetails?.isOptimisticPersonalDetail ? undefined : payeeTooltipDetails}
+                />
+            ),
+            shouldShow: isTypeSplit && action === CONST.IOU.ACTION.CREATE,
```

Comment:
Let's change this to `false` before merging this. I'll turn it back to this condition in my PR.